src/Stratis.Bitcoin.Features.MemoryPool/BlocksDisconnectedSignaled.cs

```diff
@@ -22,11 +23,11 @@ public class BlocksDisconnectedSignaled : SignalObserver<Block>
             this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
         }
 
-        protected override void OnNextCore(Block block)
+        protected override void OnNextCore(ChainedHeaderBlock block)
         {
-            this.logger.LogTrace("({0}:'{1}')", nameof(block), block.GetHash());
+            this.logger.LogTrace("({0}:'{1}')", nameof(block), block.Block.GetHash());
```

Comment:
ChainedHeaderBlock overrides .ToString()
log the whole ChainedHeaderBlock, no need for `block.Block.GetHash()`